packages/frontend/src/services/index.ts

```diff
@@ -6,3 +6,10 @@ export const apiService = axios.create({
   baseURL,
   timeout: 30 * 1000,
 });
+
+export const configureApi = (token: string) => {
```

Comment:
jak tak konfigurujesz funckje to to na górze nie jest ci potrzebne tu dodaj base url i timout a to z góry do usunięcia i nazwij to jakoś normalnie bo ta funkcja nie jest do konfiguracji api, może być tak jak wyzej apiService